<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neural Networks</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Neural Net</string>
         </attribute>
         <widget class="QCustomPlot" name="widget_h" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>511</width>
            <height>471</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="gameLayout_3"/>
         </widget>
         <widget class="QWidget" name="layoutWidget_5">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>10</y>
            <width>265</width>
            <height>471</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="setLayout_4">
           <item>
            <layout class="QHBoxLayout" name="fileLayout_3">
             <item>
              <widget class="QPushButton" name="loadQGP">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Load Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="test">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Analyze</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileLayout_5">
             <item>
              <widget class="QSpinBox" name="trainNumber">
               <property name="maximum">
                <number>8000</number>
               </property>
               <property name="value">
                <number>800</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Events for Training</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileLayout_6">
             <item>
              <widget class="QSpinBox" name="testNumber">
               <property name="maximum">
                <number>2000</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Events for Test</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileLayout_7">
             <item>
              <widget class="QSpinBox" name="spinBox_3">
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Quantity of Epochs</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileLayout_8">
             <item>
              <widget class="QSpinBox" name="spinBox_4">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>128</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Batch size</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileLayout_9">
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Neural Net Mode</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fileLayout_11">
             <item>
              <widget class="QComboBox" name="comboBox_3">
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Draw</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCustomPlot" name="widget_l" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>50</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
